flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/calcite/FlinkSqlCallBinding.java

```diff
@@ -77,20 +76,15 @@ public List<SqlNode> operands() {
 
     @Override
     public RelDataType getOperandType(int ordinal) {
-        return isNamedArgument() && !argumentTypes.isEmpty()
+        return isNamedArgument()
                 ? ((SqlOperandMetadata) getCall().getOperator().getOperandTypeChecker())
                         .paramTypes(typeFactory)
                         .get(ordinal)
                 : super.getOperandType(ordinal);
     }
 
     public boolean isNamedArgument() {
-        for (SqlNode operand : getCall().getOperandList()) {
-            if (operand != null && operand.getKind() == SqlKind.ARGUMENT_ASSIGNMENT) {
-                return !getArgumentTypes().isEmpty();
-            }
-        }
-        return false;
+        return !argumentTypes.isEmpty();
```

Comment:
If the arugment types is empty, it means the function's parameter is fixed comparing to var-length parameters. At this point, I think it's better to align the concepts with calcite. What about isFixedParameters here and rename argumentTypes to fixedArgumentTypes?

Reply:
Updated.